content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: kind
+description: Instructions to setup kind for Istio.
+weight: 21
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare kind for Istio installation.
+
+## Prerequisites
+
+- Please use latest Go version, ideally Go 1.13 or greater.
+- To use kind, you will also need to [install docker](https://docs.docker.com/install/).
+- Install the latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+## Installation steps
+
+1.  Create cluster with the following command:
+
+    {{< text bash >}}
+    $ kind create cluster --name istio-testing
+    {{< /text >}}
+
+    `--name` is used to assign specific name to the cluster. By default, the cluster will be given the name `kind`.
+
+1.  When you want to see the list of clusters, use following command:
+
+    {{< text bash >}}
+    $ kind get clusters
+    kind-istio-testing
+    {{< /text >}}
+
+    {{< tip >}}
+    `kind` is automatically prefixed in cluster name `kind-istio-testing`
+    {{< /tip >}}
+
+1.  In order to interact with a specific cluster, you only need to specify the cluster name as a context in kubectl:
+
+    {{< text bash >}}
+    $ kubectl cluster-info --context kind-istio-testing
+    Kubernetes master is running at https://127.0.0.1:32773
+    KubeDNS is running at https://127.0.0.1:32773/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    {{< /text >}}
+
+1.  In case you already have another local cluster, you must set kind as current cluster in order to use it.
+    View local cluster configuration using following command
```

Comment:
I'm not sure of the flow here. The first sentence is you must set kind as current cluster. Second sentence is viewing current context. Next paragraph (still same number) is a repeat of the first about setting current context and how to do it.  Maybe just remove the second sentence or put it under another bullet. 

Result is a bullet on setting the current context to kind, and another bullet on viewing context. 

That said, I'm not sure the viewing context bullet would really be required, or could it maybe be exchanged with a bullet on getting a list of contexts: `kubectl config get-contexts` .

Reply:
Sorry I'm worst in writing docs. Updated to include `kubectl config get-contexts`. this really shows how many local running clusters are present and also reflect cluster name properly.